fashrevwall/settings.py

```diff
@@ -1,118 +0,0 @@
-"""
```

Comment:
I thought the settings had to be in a single file, but much better to make it into a module with different files! Thanks :)